bedrock/privacy/templates/privacy/notices/websites.html

```diff
@@ -5,7 +5,7 @@
 {% extends "privacy/notices/base-notice-paragraphs.html" %}
 
 {% set body_id = "privacy-websites" %}
-{% block body_class %}mzp-t-mozilla format-none{% endblock%}
+{% block body_class %}{{ super() }} mzp-t-mozilla{% endblock%}
```

Comment:
The` {{ super() }}` here is pulling in the mzp-t-firefox class and changing the formatting from the Mozilla to the Firefox theme.